### ISSUE TITLE
Japanese-A-N on steam?

### ISSUE THREAD

**Issue by Blantas** [TyHamilton/Japanese-A-N#1]
Hi, I know probably I am very late to all this, but I was wondering, would it be possible to buy a copy of this game directly from you @TyHamilton ? I would happily donate for it. I'm preserving unavailable games on steam and Japanese-A-N is the one I missed to grab it while it was on sale.

**Comment by TyHamilton** [comment]
You want to buy a copy of my game to preserve?

Thanks,

Tyler M Hamilton

**Comment by Blantas** [comment]
> You want to buy a copy of my game to preserve? Thanks, Tyler M Hamilton

Exactly. It's just a small hobby of mine I am doing in my free time. Whenever I see a game getting removed from steam, I try to look for a copy. Most of the times the only way is contacting developers directly. 

p.s. I tried to contact you elsewhere, but I think this is the only place where you noticed. (sorry for all the spam)

**Comment by TyHamilton** [comment]
I guess my question is what is the purpose of preservation. Is it resale or
giving it out? As well what are you wanting just a Windows copy?
Thanks,

Tyler M Hamilton

**Comment by Blantas** [comment]
No, it's for personal use. I would love to keep it in my steam library. This is my profile: https://steamcommunity.com/id/Karolis_92/

Every game I acquire I redeem by myself. As you can see, I own a few games already (real count is around 26k, steam doesn't count all games towards the main game count on profile).

![image](https://github.com/TyHamilton/Japanese-A-N/assets/24192680/784aceda-c1e2-48dc-8f15-afa4846f6d11)

If you are worried about resale, I could accept a review copy too (it can be send directly from steamworks to my empty curator which I use for such cases), it doesn't require generating/sending any steam keys. I directly click a button and game gets added to my library.

**Comment by Blantas** [comment]
Or would that be not possible?

**Comment by Blantas** [comment]
If you're unsure how to send your game since it's retired on steamworks or if money is the problem, let me know.
